File: src/main/java/net/glasslauncher/mods/glassguis/screen/slot/BigHitboxSlot.java
package net.glasslauncher.mods.glassguis.screen.slot;

import net.minecraft.inventory.Inventory;
import net.minecraft.screen.slot.Slot;

public class BigHitboxSlot extends Slot implements GlassSlot {

    public BigHitboxSlot(Inventory inventory, int index, int x, int y) {
        super(inventory, index, x, y);
    }

    @Override
    public int getWidth() {
        return 24;
    }

    @Override
    public int getHeight() {
        return 24;
    }

    @Override
    public boolean shouldScaleItem() {
        return true;
    }
}

File: src/main/kotlin/net/glasslauncher/mods/glassguis/BigHitboxSlot.kt
package net.glasslauncher.mods.glassguis

import net.minecraft.inventory.Inventory
import net.minecraft.screen.slot.Slot

open class BigHitboxSlot(inventory: Inventory, index: Int, x: Int, y: Int) : Slot(inventory, index, x, y),
    CustomSizeSlot {
    override fun getWidth(): Int {
        return 24
    }

    override fun getHeight(): Int {
        return 24
    }

    override fun shouldScaleItem(): Boolean {
        return true
    }
}